tools/accuracy_checker/accuracy_checker/annotation_converters/ade20k_image_translation.py

```diff
@@ -0,0 +1,67 @@
+"""
+Copyright (c) 2018-2020 Intel Corporation
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from ..config import PathField
+from ..representation import ImageProcessingAnnotation
+from .format_converter import BaseFormatConverter, ConverterReturn
+
+
+class ADE20kImageTranslationConverter(BaseFormatConverter):
+    __provider__ = 'ade20k_image_translation'
+    annotation_types = (ImageProcessingAnnotation,)
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'images_dir': PathField(
+                optional=False, is_directory=True,
+                description="Path to directory with images."
+            ),
+            'annotations_dir': PathField(
+                optional=False, is_directory=True,
+                description="Path to directory with masks."
+            ),
+            'reference_file': PathField(
+                optional=False, description="Path to .txt file with pairs (validation:train)."
+            )
+        })
+        return parameters
+
+    def configure(self):
+        self.image_dir = self.get_value_from_config('images_dir')
+        self.annotation_dir = self.get_value_from_config('annotations_dir')
+        self.reference_dict = self.get_value_from_config('reference_file')
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        content_check_errors = [] if check_content else None
```

Comment:
I do not see any code related with content existence check inside converter, do you plan to handle it?
If not, then I think you can directly set None in ConverterReturn tuple